webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -189,7 +189,7 @@ export default class CreateIssueModal extends PureComponent {
                     type='input'
                     required={true}
                     disabled={false}
-                    maxLength={65}
+                    maxLength={120}
```

Comment:
Hi @haardikdharma10, can we change this to GitHub's limit of 256?

Reply:
Sure thing @mickmister!